admin/src/dataProvider.ts

```diff
@@ -0,0 +1,27 @@
+import { fetchUtils } from "react-admin";
```

Comment:
If you use Postgrest, you should not have to write the data provider yourself https://marmelab.com/react-admin/DataProviderList.html